lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -211,6 +212,16 @@ class FloatingAxisArtistHelperRectilinear(_FloatingAxisArtistHelperBase):
 
     def __init__(self, axes, nth_coord,
                  passingthrough_point, axis_direction="bottom"):
+        """
+
+        Parameters
+        ----------
+        axes : `mpl_toolkits.axisartist.axislines.Axes`
+        nth_coord : {0, 1}, optional
+            Along which coordinate the value varies in 2D. 0: x-axis, 1: y-axis.
+        passingthrough_point :
```

Comment:
Rename to "value" for consistency with the parent class?

Reply:
I was considering the other way around, since `passingthrough_point` gives some information about what it is used for. But with a good description (which it currently lacks...), value is probably easier to remember/spell etc.

Reply:
I think it would be much simpler to document `nth_coord` and `value` (or `passingthrough_point`, which is not even really well named as it's a scalar value, not a 2d point) together as something like "`nth_coord = 0; value = v` means to draw an axis at `x=v`; `nth_coord = 1; value = v` means to draw an axis at `y=v`".  (By the way this suggests that your wording for nth_coord's doc is also "wrong"?)